packages/@react-spectrum/dnd/docs/useDragHooks.mdx

```diff
@@ -0,0 +1,166 @@
+
+## Introduction
+
+Drag and drop is an operation performed when a user "grabs" a UI element via their pointing device and drops it on a different location or element.
+It has become a common interaction in many modern applications due to its flexibilty across a variety of use cases but can come saddled with
+a variety of accessibility issues, including but not limited to keyboard interaction parity and difficulties mimicking drag operations with mouse emulators.
+React Spectrum components that support drag and drop operations seek to address these issues by providing full keyboard and screen reader drag and drop
+support.
+
+If you are interested in adding drag and drop support to your own components, feel free to check out the [drag and drop react-aria hooks](https://www.npmjs.com/package/@react-aria/dnd)!
+
+## API
+
+<FunctionAPI function={docs.exports.useDragHooks} links={docs.links} />
+
+## How to use
+
+Enabling drag and drop for a supported React Spectrum component differs slightly from the typical event handler prop pattern that you maybe familiar with.
+Instead of providing each event handler directly to the component, you must first import `useDragHooks` from `@react-spectrum/dnd` package and provide it with
+your desired options. `useDragHooks` will then provide you with `DragHooks` that you can pass to the component via the `dragHooks` prop.
+
+You may be wondering why we didn't simply import the drag and drop hooks within the components themselves. The reason behind this is to reduce the bundle size of
+the component for users who don't need the drag and drop support for their component.
+
+The following list shows which components currently support drag and drop:
+-  [ListView](ListView.html)
+
+### Example
+
+The example below illustrates how one would use `useDragHooks` to enable drag and drop for a ListView. Note how only non-folder type items are made draggable via
+the `allowsDraggingItem` option provided to `useDragHooks`. The data transferred during the drop operation is specified by the value returned by `getItems`, in this case
+returning plain text matching the text value associated with the dragged row. Finally, the `onDragStart`, `onDragMove`, `onDragEnd` event handlers just need to be provided
+to `useDragHooks` and ListView will handle attaching those event handlers to each draggable row.
+
+To perform a drag and drop operations via a keyboard, first select the items to be dragged. You can then start the drag operation by focusing the drag handle on any of the
+selected rows and hitting <Keyboard>Enter</Keyboard>/<Keyboard>Space</Keyboard>. Once a drag operation is started, <Keyboard>Tab</Keyboard> can be used to cycle through valid
+drop targets. Hitting <Keyboard>Enter</Keyboard> will then confirm the drop operation on the focused drop target. For screen readers, please follow the instructions announced
+when focusing the row's drag handle to begin a drag operation.
```

Comment:
Felt it was important to have a brief run down of how dnd works with keyboard/screenreaders. The screen reader bit is kinda vague since the instructions are all announced and the physical steps are determined on the interaction method.